ui/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala

```diff
@@ -49,9 +49,9 @@ class SettingsResources(config: FeatureTogglesConfig,
 
 @JsonCodec case class EnvironmentAlert(content: String, cssClass: String)
 
-@JsonCodec case class CommentSettings(matchExpression: String, link: String)
+@JsonCodec case class CommentSettings(substitutionPattern: String, substitutionLink: String)
 
-@JsonCodec case class DeploySettings(requireComment: Boolean)
+@JsonCodec case class DeploySettings(validationPattern: String, exampleComment: String)
```

Comment:
I think right now we don't need it?

Reply:
thanks, removed it

Reply:
Bu right now we haven't return it to FE :) So.. We need just DeploymentConfig => something which will map config file to DTO.